### PR TITLE
Basic mods for full audit

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-7b4445b721/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a2d36fb3be/how-to/)

**Comment by catlook** [comment]
Steph...need an edit please!

File: content/general/understanding-the-cloud-computing-stack-Saas-Paas-Iaas.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+This article explains the different types of cloud computing services commonly referred to as software as a service (SaaS), platform as a service (PaaS), and infrastructure as a service (IaaS). It also provides some guidance on situations where particular flavors of cloud computing are (or are not) the best option for an organization.
 
 ### The cloud computing stack
 
-Cloud computing is often described as a stack, referring to the broad range of services built on top of one another under the name _cloud_. The generally accepted definition of _cloud computing_ comes from the National Institute of Standards and Technology (NIST). The [NIST definition](http://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-145.pdf) runs to several hundred words but begins as follows:
+Cloud computing,often described as a stack, has a broad range of services built on top of one another under the name _cloud_. The generally accepted definition of _cloud computing_ comes from the National Institute of Standards and Technology (NIST). The [NIST definition](http://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-145.pdf) runs to several hundred words but begins as follows:
```
Comment (stephamon) [inline content/general/understanding-the-cloud-computing-stack-Saas-Paas-Iaas.md]: Missing space after the "Cloud computing,"

File: content/general/understanding-the-cloud-computing-stack-Saas-Paas-Iaas.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 
 `Cloud computing` is a model for enabling ubiquitous, convenient, on-demand network access to a shared pool of configurable computing resources (e.g., networks, servers, storage, applications, and services) that can be rapidly provisioned and released with minimal management effort or service provider interaction.
```
Comment (stephamon) [inline content/general/understanding-the-cloud-computing-stack-Saas-Paas-Iaas.md]: Maybe indent this so it's clear that this is the quoted part? Block quote tags might work.

File: content/general/understanding-the-cloud-computing-stack-Saas-Paas-Iaas.md [hunk 3]
```diff
@@ -26,12 +26,12 @@ This means that end users can use parts of bulk resources and that these resourc
...
 
 - On-demand self-service: End users can sign up and receive services without the long delays that have characterized traditional IT.
-- Broad network access: Users can access the services via standard platforms (such as desktop, laptop, and mobile).
+- Broad network access: Users can access the services by using standard platforms (such as desktop, laptop, and mobile).
 - Resource pooling: Resources are pooled across multiple customers.
 - Rapid elasticity: Capability can scale to cope with demand peaks.
 - Measured service: Billing is metered and delivered as a utility service.
 
-To maximize the benefits of cloud computing, a solution must demonstrate these particular characteristics. In recent years, some traditional software vendors have marketed solutions as cloud computing that are generally accepted not to fall within this definition, a practice known as _cloud-washing_.
+To maximize the benefits of cloud computing, a solution must demonstrate these particular characteristics. In recent years, some traditional software vendors have marketed solutions as cloud computing that do not to fall within this definition, a practice known as _cloud-washing_.
```
Comment (stephamon) [inline content/general/understanding-the-cloud-computing-stack-Saas-Paas-Iaas.md]: ...cloud computing that do not fall within this definition... (extra "to" between not and fall)

File: content/general/understanding-the-cloud-computing-stack-Saas-Paas-Iaas.md [hunk 4]
```diff
@@ -43,17 +43,17 @@ Note: Although this article clearly distinguishes SaaS, PaaS, and IaaS, the diff
```
Comment (stephamon) [inline content/general/understanding-the-cloud-computing-stack-Saas-Paas-Iaas.md]: This is for line 42, which it won't let me comment on: format the note as bold so it's in the callout box when published

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-fea158a877/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-1c7e81194c/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-9a166feef8/how-to/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-098e4d8288/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-64f438c039/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9bfc72a417/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9f6581feae/how-to/)

**Pull request merged**